test/e2e/prune_test.go

```diff
@@ -57,6 +56,7 @@ var _ = Describe("Podman rm", func() {
 	})
 
 	It("podman image prune none images", func() {
+		SkipIfRemote()
```

Comment:
Is this test making a "<none>" image?  I think so, but am not 100% sure.  Can we add a check to validate that there is a non-dangling image still when it's complete?

Reply:
ugh, s/b "making a `<none>` image"

Reply:
Yes this builds something that leaves behind dangling images (none ones).  Added an expect to make sure named, unused images are not deleted.